src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -2213,6 +2213,16 @@ SizeInMegaBytesFormat = {0} MB
 SizeInGigaBytesFormat = {0} GB
 ;Size in TeraBytes format
 SizeInTeraBytesFormat = {0} TB
+; display string for the operator cost property - 0.###### - is the float number format specifier
+OperatorDisplayCost(double cost, int percentage) = {0:0.#######} ({1}%)
+;This node name will be visible in Active Accesibility
+NodeDisplayPropertiesName2(string title, long nodeId) = {0}[{1}]
+;This node name will be visible in Active Accesibility
+NodeDisplayPropertiesName1(long nodeId) = [{0}]
```

Comment:
These don't seem like things that actually need to be localized - there's no text to localize here

Reply:
I got these strings from SSMS. Should I move them somewhere else? I see that they have added some basic formated texts in .strings file.

Reply:
I would just embed them directly into the code using string interpolation or string.Format